unit-9/exercise-5.js

```diff
@@ -0,0 +1,16 @@
+// a, b lần lượt là điểm thi giữa kỳ, điểm thi cuối kỳ
```

Comment:
please in english
Và, đặt tên khác cho a, b. Không đặt tên biến tối nghĩa như vậy

vd:
middleSeasonScore
lastSeasonScore